### PR TITLE
common: add copy-test-files target

### PR DESCRIPTION
**Pull request by blazej-smorawski**

### CODE REVIEW

File: utils/copy.py [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
+# Optional positional argument
+parser.add_argument('--regex', type=str,
+                    help='Use regural expression to choose files.')
 
-# get absolute input and output paths
-input_path = os.path.join(
-    os.getenv('MESON_SOURCE_ROOT'),
-    os.getenv('MESON_SUBDIR'),
-    sys.argv[1])
+parser.add_argument('--root-directory', type=str,
```
Comment (kswiecicki) [inline utils/copy.py]: 'help' should describe a '--root-directory' argument, not the regex one.

File: utils/copy.py [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
 
-import os, sys, shutil
+# Instantiate the parser
+parser = argparse.ArgumentParser(description='Copy files tree from $MESON_SOURCE_ROOT to $MESON_BUILD_ROOT')
 
-copy_prefixes = sys.argv[3]
-copy_suffixes = sys.argv[4]
+# Optional positional argument
+parser.add_argument('--regex', type=str,
+                    help='Use regural expression to choose files.')
```
Comment (kswiecicki) [inline utils/copy.py]: 'regular', not 'regural'

Comment (blazej-smorawski) [reply utils/copy.py]: Done

File: utils/copy.py (hunk 1 above)
Comment (blazej-smorawski) [reply utils/copy.py]: Done

**Pull request closed without merging**